asdf/tagged.py

```diff
@@ -35,9 +35,9 @@
 
 import six
 
+from astropy import time
```

Comment:
I haven't looked at the PR but could you place this import statement somewhere in a function or class so that it's not always required.